web/lib/components/flashcards/useAllSets.js

```diff
@@ -2,14 +2,24 @@ import { useQuery, useMutation, useQueryClient } from 'react-query'
 
 import {
   getAllSets,
+  getPaginatedSets,
   upsertSet,
   upsertFlashcard,
   deleteFlashcardSet,
 } from './query'
 
-export function useAllSets({ user, fetchAllSets = false }) {
+function getRange(page, perPage) {
+  return { from: (page + 1) * perPage - perPage, to: (page + 1) * perPage - 1 }
```

Comment:
The logic here is weird? Clicking Next goes to page 1, Prev goes to page 2?

Reply:
this function return range of set.
e.g:  if page is 0, then `from` should be 0 and `to` would be 8
the calculation here does not reflect the page number.